recipes/libmicrohttpd/all/conanfile.py

```diff
@@ -0,0 +1,176 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir
+from conan.tools.microsoft import MSBuild, MSBuildToolchain, is_msvc, vs_layout
+from conan.errors import ConanInvalidConfiguration
+from conans import AutoToolsBuildEnvironment
+import functools
+import os
+
+required_conan_version = ">1.48.0"
```

Comment:
```suggestion
required_conan_version = ">1.52.0"
```
export_conandata_patches requires 1.52.0

Reply:
Isn't this `required_conan_version` thing broken?
Because `export_conandata_patches` has already been imported by this point.
If that function is not available due to an older conan version, then this point is never reached due to an `ImportError` being thrown.

Reply:
[Never mind, conan has this covered](https://github.com/conan-io/conan/blob/8936c117db983e64e8c43a325a4759f54cdf1715/conans/client/loader.py#L488)